### DISCUSSION TITLE
VSCode MicroPico - MicroPico modules not found

### DISCUSSION THREAD

**Discussion by Karl60** [paulober/MicroPico#235] [Q&A]
MicroPico version 3.8.2
VSCode Version 1.91.1
In short, all was working well 2 days ago but now all MicroPico-specific imports, eg, import machine, fail with a module not found error.  
MicroPico was uninstalled and reinstalled, no change.  MicroPico project reinitialised, no change.
I'm at a loss as to what's changed.  I presume I've unintentionally changed some settings somewhere.
Any guidance on how to fault-find this problem would be appreciated.

**Comment by mogplus88** [comment]
Three months later I found the same problem. Last time I tried it (which was admittedly about two years ago) MicroPico worked a treat. Now, no import libraries are recognised. I wanted to try it with ESP32, but that didn't seem to work either. I understand it's experimental so I wasn't expecting miracles, but I thought I could at least import a module (like esp32), but it failed with the same error. I'll check back again in another couple of months.
It might not be micropico. I just tried rt-thread, which I've used successfully before, and that doesn't seem to work either... It will import, but then generates other odd messages.
